test/app_test.dart

```diff
@@ -62,11 +62,89 @@ Future<void> main([List<String>? args]) async {
     final app = App(configuration);
     final credentials = Credentials.anonymous();
     final user = await app.logIn(credentials);
+    expect(user.state, UserState.loggedIn);
   });
+
   test('Application get all users', () {
     final configuration = AppConfiguration(generateRandomString(10));
     final app = App(configuration);
     var users = app.users;
     expect(users.isEmpty, true);
   });
+
+  baasTest('App log out no current user is no operation and does not crash', (configuration) async {
+    final app = App(configuration);
+    await app.logout();
+  });
+
+  baasTest('App log out user', (configuration) async {
+    final app = App(configuration);
+    final user = await app.logIn(Credentials.emailPassword(testUsername, testPassword));
+
+    expect(user.state, UserState.loggedIn);
+    await app.logout(user);
+    expect(user.state, UserState.loggedOut);
+  });
+
+  baasTest('App remove user', (configuration) async {
+    final app = App(configuration);
+    final user = await app.logIn(Credentials.emailPassword(testUsername, testPassword));
+
+    expect(user.state, UserState.loggedIn);
+    await app.removeUser(user);
```

Comment:
Is it expected after removing a user this user to disappear from cloud? I tested it and the user is still existing in the app.
```dart
baasTest('App remove user', (configuration) async {
    final app = App(configuration);
    final user = await app.logIn(Credentials.emailPassword(testUsername, testPassword));

    expect(user.state, UserState.loggedIn);
    await app.removeUser(user);
    expect(user.state, UserState.removed);
    final user1 = await app.logIn(Credentials.emailPassword(testUsername, testPassword));
    expect(user.id, user1.id);
  });```

Reply:
`removeUser` only removes the user from the local app, doesn't delete them in cloud.